docker/debug/Dockerfile

```diff
@@ -5,12 +5,19 @@ FROM golang:1.23.0-alpine AS build
 ARG TARGETARCH
 ENV GOPATH /go
 RUN apk add --update --no-cache ca-certificates make git build-base mailcap
+
+# Copy go.mod and go.sum files
+COPY go.mod go.sum /go/src/debug-delve/
+WORKDIR /go/src/debug-delve
+# Download dependencies (including Delve)
+RUN go mod download
```

Comment:
I think you want this command to only run before L20, inside the if statement (otherwise it might fail on unsupported architectures)

Reply:
also, you can do `cd /go/src/debug-delve && go mod download` instead of using WORKDIR